kolibri/plugins/facility_management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -292,10 +292,15 @@
         }
       },
       focusOnInvalidField() {
-        this.nameIsInvalid && this.$refs.name.focus();
-        this.usernameIsInvalid && this.$refs.username.focus();
-        this.passwordIsInvalid && this.$refs.password.focus();
-        this.confirmedPasswordIsInvalid && this.$refs.confirmedPassword.focus();
+        if (this.nameIsInvalid) {
+          this.$refs.name.focus();
+        } else if (this.usernameIsInvalid) {
+          this.$refs.username.focus();
+        } else if (this.passwordIsInvalid) {
+          this.$refs.password.focus();
+        } else if (this.confirmedPasswordIsInvalid) {
+          this.$refs.confirmedPassword.focus();
+        }
```

Comment:
nice job improving the readability here

Reply:
Looks like the intent may have actually been to show all errors simultaneously. Could you confirm whether this is the case and which version is the desired behavior?

Reply:
Yes that behavior still exists
![wut](https://user-images.githubusercontent.com/7193975/41802612-da8a2fc0-7636-11e8-9998-69d40b6b43d2.gif)